db/migrate/20150212124533_add_shipping_enabled_to_community.rb

```diff
@@ -0,0 +1,5 @@
+class AddShippingEnabledToCommunity < ActiveRecord::Migration
+  def change
+    add_column :communities, :shipping_enabled, :integer, default: 0
```

Comment:
I'm not sure if it's good idea to put this into community table even temporarily. I spent considerable amount of time moving the paypal enabled out of community table after the first release when the check had been embedded already into a lot of places in the code base. What about marketplace_settings or listing_settings or selling_settings or something like that, per community table?